python/py-llfuse/Portfile

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
 
 PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-llfuse
 version             1.1.1
+revision            1
```

Comment:
No need to increase the revision for this. Revision only has to be increased if the installed files are changed.